.github/labeler.yml

```
@@ -67,6 +67,7 @@
   - tools/*
 "Category: Mod Cog":
   - redbot/cogs/mod/*
+  - docs/cog_guides/mod.rst
```

For consistency with the other entries in the labeler file (the one in the branch this PR is created from is outdated so you can't really see that's the case), there should be appropriate comments:
```suggestion
"Category: Mod Cog":
  # Source
  - redbot/cogs/mod/*
  # Docs
  - docs/cog_guides/mod.rst
```